### PR TITLE
[site-server] Add a production server

### PR DESCRIPTION
**Pull request by aomarks**
Just a minimal static Koa server

Also fixes the SPDX identifiers in client .ts files to be Apache-2.0 instead of BSD-3-Clause

Fixes https://github.com/webcomponents/webcomponents.org/issues/1326

### CODE REVIEW

File: packages/server/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@webcomponents/internal-site-server",
```
Comment (justinfagnani) [inline packages/server/package.json:2]: I would probably name the folder `site-server` to be more clear when browsing the FS and also seeing `catalog-server`

File: packages/server/src/prod-server.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import Koa from 'koa';
+import koaStatic from 'koa-static';
+import koaConditionalGet from 'koa-conditional-get';
+import koaEtag from 'koa-etag';
+import {fileURLToPath} from 'url';
+import * as pathlib from 'path';
+
+const PORT = process.env['PORT'] || 8080;
+const STATIC_ROOT = pathlib.resolve(
+  fileURLToPath(import.meta.url),
```
Comment (justinfagnani) [inline packages/server/src/prod-server.ts:16]: super nit: might be nice to use `fileURLToPath(new URL('../../../context/_site', import.meta.url))`

**Review by justinfagnani — APPROVED** [review]

File: packages/server/package.json (hunk 1 above)
Comment (aomarks) [reply packages/server/package.json:2]: Done

File: packages/server/src/prod-server.ts (hunk 2 above)
Comment (aomarks) [reply packages/server/src/prod-server.ts]: Done (removes need for path library import :-1: )

**Pull request merged**